src/components/svg/SvgImage.js

```diff
@@ -175,13 +172,9 @@ class SvgImage extends Component {
             showsHorizontalScrollIndicator={false}
             showsVerticalScrollIndicator={false}
             source={{ html }}
-            style={[
-              styles,
-              props.style,
-              { display: this.state.loaded || android ? 'flex' : 'none' },
```

Comment:
Is this safe to get rid of?

Reply:
ye: https://cloud.skylarbarrera.com/Screen-Recording-2022-11-28-14-59-52.mp4